### PR TITLE
Prc

### PR DESCRIPTION
**Pull request by keithamus**
Reprovisioned, and dropped CSS deps in favour of moving them upstream.

Marked as breaking due to dropping CSS deps.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,69 +1,68 @@
...
-      0.5em *
-      var(--text-line-height-sans-on-step--2)
-    );
+    top:
+      calc(
+        var(--cookiemessage-min-height, 64px) / 2 -
+        var(--cookiemessage-vertical-padding, var(--grid-spacing-sheep)) -
+        0.5em *
+        var(--cookiemessage-line-height, var(--text-line-height-sans-on-step--2))
+      );
```
Comment (fabiosantoscode) [inline src/index.css]: Does this compile as expected? There are enormous issues with cssnext whitespace, and the lint forces you to trip on them.

Comment (keithamus) [inline src/index.css]: compiles down to:

```css
    top: calc(
        64px / 2 -
        1.45em -
        0.5em *
        1.34
      );
```

For some reason the calc isn't resolved to a whole value, which is odd. But calc is well supported so not the end of the world.

Comment (fabiosantoscode) [inline src/index.css]: Not really the end of the world, no. But still, it's best to have things compile as they should. It's not like the new version inherently better than the old one, so it doesn't really justify having this error.

Comment (keithamus) [inline src/index.css]: The whitespace does not make a difference to the output. It won't compile any further due to `em` being used which is a relative measurement. I can wrap parts of it in calc to reduce the number of params in the output but whitespace (the only thing changed) is not the issue here.

**Comment by keithamus** [comment]
This will likely fail because component-typography needs fix the issues with react-simpletabs not supporting React 15.

**Pull request merged**